app/views/shared/_dates_and_assignment_header.html.erb

```diff
@@ -1,74 +1,23 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds" id="dates-and-assignment-details">
-    <p class="govuk-body">
-      <strong>Validation date:</strong>
-      <% if @planning_application.valid_from.present? %>
-        <%= time_tag @planning_application.valid_from %>
-      <% else %>
-        Not yet valid
+    <% if @planning_application.consultation.present? && @planning_application.consultation.end_date.present? %>
```

Comment:
I think this line could be

```suggestion
    <% if @planning_application.consultation&.end_date %>
```

We don't need `present?` because it can only be set or `nil`, there is no 'not present' value other than nil. And if we use `&.` we don't need to check separately for its parent object.

Reply:
Good point, copied the previous behaviour without trying to improve it